packages/gestalt/src/Dropdown.js

```diff
@@ -83,16 +83,38 @@ const renderChildrenWithIndex = (childrenArray) => {
   }, []);
 };
 
-type IdealDirection = 'up' | 'right' | 'down' | 'left';
-
 type Props = {|
+  /**
+   * Ref for the element that the Dropdown will attach to, will most likely be a [Button](/button). See the [Accessibility](https://gestalt.netlify.app/dropdown#Accessibility) guidelines to learn more.
+   */
   anchor?: ?HTMLElement,
```

Comment:
This `?` is getting lost for some reason. Do you know why?

Reply:
I'll get fixed when merging Popover changes.